content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+ ---
+reviewers:
+- caesarxuchao
+- dchen1107
+title: नोड्स
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes _Nodes_ पर चलने के लिए पॉड्स में कंटेनर रखकर आपका {{< glosary_tooltip text="workload" term_id="workload" >}} चलाता है।
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.